tests/client/core/test_store.js

```diff
@@ -1,21 +1,55 @@
 import { middleware } from 'core/store';
 
 describe('core store middleware', () => {
-  it('includes the middleware in development', () => {
-    const config = {
-      get() {
-        return true;
+  function devConfig(isDevelopment) {
+    return {
+      get(key) {
+        return (key === 'isDevelopment') ? isDevelopment : false;
       },
     };
-    assert.isFunction(middleware({ __config: config }));
+  }
+
+  it('includes the middleware in development', () => {
+    const __createLogger = sinon.spy(f => f);
+    assert.isFunction(middleware({
+      __config: devConfig(true), __createLogger,
```

Comment:
This would be better passed via an object so the intent of the boolean value is more obvious.